acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/tls/ClientSideTlsAcceptanceTest.java

```diff
@@ -128,7 +133,7 @@ void ethSignerProvidesSpecifiedClientCertificateToDownStreamServer(@TempDir Path
     final HttpServer web3ProviderHttpServer =
         serverFactory.create(serverCert, ethSignerCert, workDir);
 
-    final Signer signer =
+    signer =
```

Comment:
Doesn't need to be assigned to the field signer as we are already doing that in createAndStartSigner.

Reply:
oops - that would be bad.